provisioning/service/samples/ProvisioningServiceBulkOperation/Program.cs

```diff
@@ -85,18 +85,17 @@ public static void Main(string[] args)
                 #endregion
 
                 #region Query info of individualEnrollment
-                //Console.WriteLine("\nCreating a query for enrollments...");
-                //QuerySpecification querySpecification =
-                //        new QuerySpecificationBuilder("*", QuerySpecificationBuilder.FromType.ENROLLMENTS)
-                //                .createSqlQuery();
-                //Query query = provisioningServiceClient.createIndividualEnrollmentQuery(querySpecification);
-
-                //while (query.hasNext())
-                //{
-                //    Console.WriteLine("\nQuerying the next enrollments...");
-                //    QueryResult queryResult = query.next();
-                //    Console.WriteLine(queryResult);
-                //}
+                Console.WriteLine("\nCreating a query for enrollments...");
+                QuerySpecification querySpecification = new QuerySpecification("SELECT * FROM enrollments");
+                using (IQuery query = provisioningServiceClient.CreateIndividualEnrollmentQuery(querySpecification, 2))
```

Comment:
>2 [](start = 116, length = 1)

Just out of curiosity: what is this magic number?

Reply:
This is just a random number, to be different than 0, it is a page size, I'll define a constant with a nice name to better clarify it.

Reply:
Done